### PR TITLE
feat(sdk): support update organization's displayName

### PR DESCRIPTION
**Pull request by ahasgx**
extra fields (type, status, invite type) of organization

### CODE REVIEW

File: vortex-java-sdk/vortex-java-sdk-core/src/main/java/com/consoleconnect/vortex/core/entity/OrderEntity.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import jakarta.persistence.Index;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Getter
+@Setter
+@Table(
+    name = "white_label_order",
```
Comment (DaveXiong) [inline vortex-java-sdk/vortex-java-sdk-core/src/main/java/com/consoleconnect/vortex/core/entity/OrderEntity.java]: replace `white_label` with `vortex` pls

File: vortex-java-sdk/vortex-java-sdk-core/src/main/java/com/consoleconnect/vortex/core/entity/OrderEntity.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    indexes = {@Index(columnList = "order_id"), @Index(columnList = "organization_id")})
+public class OrderEntity extends AbstractEntity {
```
Comment (DaveXiong) [inline vortex-java-sdk/vortex-java-sdk-core/src/main/java/com/consoleconnect/vortex/core/entity/OrderEntity.java]: OrderEntity should be in `gateway` module

File: vortex-java-sdk/vortex-java-sdk-core/src/main/java/com/consoleconnect/vortex/core/entity/OrganizationEntity.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+@Entity
+@Getter
+@Setter
+@Table(
+    name = "white_label_organization",
+    indexes = {
+      @Index(columnList = "display_name", unique = true),
+      @Index(columnList = "name", unique = true)
+    })
+public class OrganizationEntity extends AbstractEntity {
```
Comment (DaveXiong) [inline vortex-java-sdk/vortex-java-sdk-core/src/main/java/com/consoleconnect/vortex/core/entity/OrganizationEntity.java]: this entity should be in `iam` module

File: vortex-java-sdk/vortex-java-sdk-core/src/main/java/com/consoleconnect/vortex/core/enums/LoginTypeEnum.java [hunk 4]
```diff
@@ -0,0 +1,6 @@
+package com.consoleconnect.vortex.core.enums;
+
+public enum LoginTypeEnum {
+  SSO,
+  INVITE
```
Comment (DaveXiong) [inline vortex-java-sdk/vortex-java-sdk-core/src/main/java/com/consoleconnect/vortex/core/enums/LoginTypeEnum.java:5]: replace `INVITE` with `USERNAME_PASSWORD` ?

File: vortex-java-sdk/vortex-java-sdk-core/src/main/java/com/consoleconnect/vortex/core/enums/OrgStatusEnum.java [hunk 5]
```diff
@@ -0,0 +1,6 @@
+package com.consoleconnect.vortex.core.enums;
+
+public enum OrgStatusEnum {
+  ACTIVE,
+  INACTIVE
```
Comment (DaveXiong) [inline vortex-java-sdk/vortex-java-sdk-core/src/main/java/com/consoleconnect/vortex/core/enums/OrgStatusEnum.java:5]: Can auth0 support `inactive` ? if not, how to handle this ?

File: vortex-java-sdk/vortex-java-sdk-iam/src/main/java/com/consoleconnect/vortex/iam/controller/MgmtOrganizationController.java [hunk 6]
```diff
@@ -29,34 +31,44 @@
...
 public class MgmtOrganizationController {
 
+  private final VortexOrganizationService vortexOrganizationService;
   private final OrganizationService service;
 
   @PreAuthorize("hasPermission('mgmt:org', 'list')")
   @Operation(summary = "List all existing organizations")
   @GetMapping("")
-  public Mono<HttpResponse<Paging<Organization>>> search(
+  public Mono<HttpResponse<Paging<OrganizationEntity>>> search(
```
Comment (DaveXiong) [inline vortex-java-sdk/vortex-java-sdk-iam/src/main/java/com/consoleconnect/vortex/iam/controller/MgmtOrganizationController.java:40]: genereally won't expose `entity` directly, better to define a dto

**Review by DaveXiong — CHANGES_REQUESTED** [review]

File: vortex-java-sdk/vortex-java-sdk-core/src/main/java/com/consoleconnect/vortex/core/entity/OrganizationEntity.java (hunk 3 above)
Comment (ahasgx) [reply vortex-java-sdk/vortex-java-sdk-core/src/main/java/com/consoleconnect/vortex/core/entity/OrganizationEntity.java]: done

File: vortex-java-sdk/vortex-java-sdk-core/src/main/java/com/consoleconnect/vortex/core/entity/OrderEntity.java (hunk 2 above)
Comment (ahasgx) [reply vortex-java-sdk/vortex-java-sdk-core/src/main/java/com/consoleconnect/vortex/core/entity/OrderEntity.java]: done

File: vortex-java-sdk/vortex-java-sdk-core/src/main/java/com/consoleconnect/vortex/core/entity/OrderEntity.java (hunk 1 above)
Comment (ahasgx) [reply vortex-java-sdk/vortex-java-sdk-core/src/main/java/com/consoleconnect/vortex/core/entity/OrderEntity.java]: done

File: vortex-java-sdk/vortex-java-sdk-core/src/main/java/com/consoleconnect/vortex/core/enums/LoginTypeEnum.java (hunk 4 above)
Comment (ahasgx) [reply vortex-java-sdk/vortex-java-sdk-core/src/main/java/com/consoleconnect/vortex/core/enums/LoginTypeEnum.java]: done

File: vortex-java-sdk/vortex-java-sdk-core/src/main/java/com/consoleconnect/vortex/core/enums/OrgStatusEnum.java (hunk 5 above)
Comment (ahasgx) [reply vortex-java-sdk/vortex-java-sdk-core/src/main/java/com/consoleconnect/vortex/core/enums/OrgStatusEnum.java]: not support

File: vortex-java-sdk/vortex-java-sdk-iam/src/main/java/com/consoleconnect/vortex/iam/controller/MgmtOrganizationController.java (hunk 6 above)
Comment (ahasgx) [reply vortex-java-sdk/vortex-java-sdk-iam/src/main/java/com/consoleconnect/vortex/iam/controller/MgmtOrganizationController.java]: done

**Review by DaveXiong — APPROVED** [review]
LGTM

**Pull request merged**